components/colorslider/stories/template.js

```diff
@@ -33,7 +33,7 @@ export const Template = ({
 					role="presentation"
 					style="background: linear-gradient(to ${vertical
 						? "bottom"
-						: "right"}, rgb(255, 0, 0) 0%, rgb(255, 255, 0) 17%, rgb(0, 255, 0) 33%, rgb(0, 255, 255) 50%, rgb(0, 0, 255) 67%, rgb(255, 0, 255) 83%, rgb(255, 0, 0) 100%);"
+						: "right"}, ${gradientStops});"
```

Comment:
Noticing that this change has resulted in the colorhandle no longer matching the color of the slider below it. 

Could we also add `--spectrum-picked-color` as an arg that could be set to match the gradient start color?

Reply:
I've added this to the stories, and have also added a new story for the "With Image" variant.